sklearn/metrics/tests/test_common.py

```diff
@@ -211,6 +224,11 @@
     "label_ranking_average_precision_score",
 ]
 
+# Those metrics don't support average=None for multiclass input
+NONE_AVERAGE_UNDEFINED_MULTICLASS = [
```

Comment:
A lot of current failures occur due to, `jaccard_similarity_score` raising `ValueError: In multiclass classification average must be one of ('micro', 'macro', 'weighted'), got average=None.`.

I intend to use:
```python
if name in NONE_AVERAGE_UNDEFINED_MULTICLASS and average is None:
    continue
```
Do you think this solution is in the right direction?